src/lib/resource/sections/applicant-select-reasons-for-the-delay-in-making-your-application.js

```diff
@@ -39,6 +42,9 @@ module.exports = {
                     meta: {
                         classifications: {
                             theme: 'crime'
+                        },
+                        summary: {
+                            title: 'Brief explanation'
```

Comment:
Should this be more specific? in isolation, "Brief explanation" doesn't mean much. The order of the questions, and the section it sits in is the only indication of what it explicitly means